src/status_im2/navigation/roots.cljs

```diff
@@ -141,14 +141,12 @@
   (merge (old-roots)
          ;;INTRO (onboarding carousel)
          {:intro-stack
-          {:root {:stack {:id       :intro
-                          :children [{:component {:name :intro
-                                                  :id :intro
-                                                  :options
-                                                  {:statusBar (merge
```

Comment:
This merge returns a map containing the `:statusBar`  key again, it is not used, so I removed it